docker/docker-compose-test.yaml

```diff
@@ -35,15 +45,29 @@ services:
         tail -f /dev/null
     volumes:
       - ${TEST_DATA_DIR}:/main/test_data
-      - ./apt_requirements.txt:/tmp/apt_requirements.txt
-      - ../../element-lab:/main/element-lab
-      - ../../element-animal:/main/element-animal
-      - ../../element-session:/main/element-session
-      - ../../element-interface:/main/element-interface
-      - ../../element-miniscope:/main/element-miniscope
+      # - ./apt_requirements.txt:/tmp/apt_requirements.txt
+      # - ../element-lab:/main/element-lab
+      # - ../element-animal:/main/element-animal
+      # - ../element-session:/main/element-session
+      # - ../element-event:/main/element-event
+      # - ../element-interface:/main/element-interface
+      # - ../element-miniscope:/main/element-miniscope
       - ..:/main/workflow-miniscope
     depends_on:
       db:
-        condition: service_healthy
+        condition: service_started
+
+  adminer:
+    networks:
+      miniscope:
+    image: adminer
+    restart: always
+    ports:
+      - 8082:8080
+
 networks:
-  main:
\ No newline at end of file
+  miniscope:
+    name: workflow-miniscope-network
+
+volumes:
+  mysql-miniscope:
```

Comment:
Hi @iamamutt - Could you remind me why `volumes` is here? I see it in the copy you drafted prior to this PR. Does `networks` not provide what we need on it's own?

Reply:
@CBroz1 That was there to map the data from the mysql database to a docker volume instead of bind mounting to the local filesystem. If you don't need access to the database data at all, you can delete the volumes section at the bottom, +72 to +73, and also it's mention at lines +19 to +20.

If you want a bind mounted volume, change `- mysql-miniscope:/var/lib/mysql` to `- path/to/some/data/dir:/var/lib/mysql`, and you can delete the volumes section at the end of the file.

Reply:
That makes sense. Thanks @iamamutt!